editor.planx.uk/src/@planx/components/shared/Preview/SummaryList.tsx

```diff
@@ -78,11 +80,19 @@ const components: {
 
 interface SummaryListsBySectionsProps extends SummaryListProps {
   sectionComponent: React.ElementType<any> | undefined;
+  showStatusTag?: boolean;
```

Comment:
will cleanup this optional prop and the "Tag" export depending if we decide design in #1505 is useful or not!

Reply:
closed #1505, removed this unused prop!